src/main/java/com/redhat/restdemo/controllers/AuthorController.java

```diff
@@ -1,36 +1,99 @@
 package com.redhat.restdemo.controllers;
 
 import com.redhat.restdemo.model.entity.Author;
+import com.redhat.restdemo.model.entity.Book;
 import com.redhat.restdemo.model.service.AuthorService;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
```

Comment:
* imports are bad idea in general. Try to configure your IDE to use separate imports instead of *.